src/mumble/Overlay.cpp

```diff
@@ -243,14 +244,8 @@ void Overlay::createPipe() {
 	pipepath = QLatin1String("MumbleOverlayPipe");
 #else
 	{
-		QString xdgRuntimePath = QProcessEnvironment::systemEnvironment().value(QLatin1String("XDG_RUNTIME_DIR"));
-		QDir xdgRuntimeDir     = QDir(xdgRuntimePath);
-
-		if (!xdgRuntimePath.isNull() && xdgRuntimeDir.exists()) {
-			pipepath = xdgRuntimeDir.absoluteFilePath(QLatin1String("MumbleOverlayPipe"));
-		} else {
-			pipepath = QDir::home().absoluteFilePath(QLatin1String(".MumbleOverlayPipe"));
-		}
+		QDir mumbleRuntimeDir = Mumble::getRuntimeDir();
+		pipepath = mumbleRuntimeDir.absoluteFilePath(QLatin1String("MumbleOverlayPipe"));
```

Comment:
This should also be encapsulated as its own function. Aka: Add another function `getOverlayPipePath` and this function's job is then to call `getRuntimeDir` and then append the pipe's name to that and return it.